src/sentry/deletions/defaults/group.py

```diff
@@ -3,12 +3,34 @@
 import os
 from sentry import eventstore, nodestore
 from sentry.eventstore.models import Event
-from sentry.models import EventAttachment, UserReport
-from sentry.reprocessing2 import delete_unprocessed_events
+from sentry import models
 
 from ..base import BaseDeletionTask, BaseRelation, ModelDeletionTask, ModelRelation
 
 
+GROUP_RELATED_MODELS = (
```

Comment:
Despite the name this is still related to group deletion and the *order* of items is specific to deletion. I was also considering duplicating the list like group merge/unmerge does. Let me know what you prefer.

Reply:
You're right, I'll leave this up to you. I found it interesting that this is being imported from reprocessing to copy over information rather than delete it, that's all.